cfgov/unprocessed/apps/paying-for-college/js/disclosures/views/expenses-view.js

```diff
@@ -148,19 +148,7 @@ const expensesView = {
       expensesView.updateView( getExpenses.values() );
       Analytics.sendEvent( getDataLayerOptions( 'Region Changed', region ) );
     } );
-  },
-
-  /**
-   * Listener for the Feedback BUTTON
-   */
-  feedbackBtnListener: function() {
```

Comment:
Looks line 20 referencing this function needs removal too https://github.com/cfpb/consumerfinance.gov/blob/9693c7e3418651998ad248ff5e08f1bba0cd8050/cfgov/unprocessed/apps/paying-for-college/js/disclosures/views/expenses-view.js#L20